src/index.js

```diff
@@ -0,0 +1,118 @@
+import './style.css';
+
+let tasks = [];
+const taskWrapper = document.querySelector('.to-dos');
```

Comment:
- [ ] Please make sure to select the correct query selectors. Right now, you have a type error because the console says `taskWrapper` is null.
Here's an image of the error:
![Captura de pantalla 2022-05-12 221430](https://user-images.githubusercontent.com/59100477/168160419-bf276e3d-aea2-4fad-85ed-e6930baff604.jpg)

- [ ] Because of this error, the tasks are not being displayed on the screen. It is being saved in the localStorage, but they are not shown in the screen, so please work on this issue.

Reply:
@Lordkaito Thanks for the feedback. 
Please can you give me more details about correct query selector and how to fix it?

Reply:
@Lordkaito Thanks for the feedback. 
Please can you give me more details about correct query selector and how to fix it?